### PR TITLE
feat(commands): add panel-mode critical review (mkarch default, mkplan opt-in)

### PR DESCRIPTION
**Pull request by isseis**
## 概要

`/mkarch`・`/mkplan` のレビュー工程に **panel モード**（マンデート分割した複数レビュアーを並列起動し、結果を合成）を追加する。共有手順 `_lib/review-subagent-pattern.md` に panel モードを定義し、各コマンドから opt-in する。

## 動機（実証ベース）

`02_architecture.md` を「アーキテクト兼 SRE」1 体（2 パスで「Critical/Major なし」と結論）と、「アーキテクト専任＋SRE 専任（マンデート分割）」2 体で**比較レビュー**したところ、分割2体は兼務が見逃した **Major 級を約13件**検出した（コンパイル不能の型誤り、AC と矛盾する再利用、TOCTOU の過大主張、fail-open、ロールアウト不在 等）。兼務は満足化（satisfice）して片側レンズが薄くなる、という仮説が支持された。

## 変更内容

- **`_lib/review-subagent-pattern.md`**: 「Panel mode」を追加。
  - **ラベルでなくマンデートを分割**（非重複の担当＋明示的 out-of-scope）。相手領域は 1 行 OUT-OF-LANE FLAG のみ。
  - 並列起動（wall-clock 増なし）→ 呼び出し側が**合成**（dedup・severity 整合・継ぎ目の横断論点を再構成）→ 既存の fix/再レビュー loop。
  - コスト（〜N× トークン）を明記し、小・低リスク成果物は単一レビュアーを既定とする。
- **`mkarch.md`**: レビューを panel（**アーキテクト＋SRE**、既定）に。architecture は高 stakes で満足化リスクが大きいため。
- **`mkplan.md`**: panel（**SWE＋テスト/SDET**、**opt-in**）。plan の自然な継ぎ目は「作る vs 検証する」なので engineer/SRE ではなく SWE/テストで分割。Conditional checks はテスト側へ。AC 追跡を両端からチェック。ルーブリックが網羅的で stakes も低めのため既定は単一、高リスク計画のみ panel。

## 設計判断

- 分割軸は**成果物固有**: architecture=architect/SRE（構造 vs 実行時）、plan=SWE/テスト（作る vs 検証）。
- 既定: mkarch=panel、mkplan=単一（opt-in panel）。

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: .claude/commands/mkplan.md [hunk 1]
```diff
@@ -67,13 +67,16 @@ Work in the following order.
...
+   - **FILES**: the implementation plan document, the architecture document, the requirements document, the requirements process guide, and the test organization guide (paths in `_context.md`), as resolved absolute-path strings. In panel mode, pass the same FILES to both reviewers.
+   - **CRITERIA**: every item from the Technical correctness checklist, the Conditional checks section, and the Readability and consistency checklist below, copied verbatim. For Conditional checks, evaluate each item that applies to the plan; mark inapplicable items N/A (N/A is not a finding). In panel mode, give both reviewers all of them as the shared floor, but each reports only items within its mandate (deferring the rest as one-line OUT-OF-LANE FLAGS); route the Conditional checks to Reviewer B.
+
+   Extra rule: commit the implementation plan document only after all review passes are complete (in panel mode, after synthesizing the two outputs per `_lib`) and all Critical and Major issues are resolved.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Added an explicit **Synthesis** input bullet to mkplan step 8, mirroring mkarch (dedup, reconcile severities, reconstruct the AC->{task,test} matrix, promote confirmed flags), instead of only mentioning it in the Extra rule. (9026c631)

File: .claude/commands/_lib/review-subagent-pattern.md [hunk 2]
```diff
@@ -50,3 +54,38 @@ After receiving findings:
...
+   Agent tool calls — so wall-clock does not increase. Each prompt is self-contained
+   (embed FILES as absolute paths; copy that reviewer's CRITERIA subset verbatim;
+   state mandate + out-of-scope; require the same Severity/Location/Problem/Suggestion
+   output format).
+3. **Synthesize before fixing.** The caller (not a subagent) merges the panel
+   outputs: **dedup** overlapping findings; **reconcile** conflicting severities
+   (take the higher); and **reconstruct cross-cutting issues** that sit in the seam
+   between mandates (a structural choice with an operational consequence) — this
+   synthesis is panel mode's main risk to mitigate, since no single reviewer sees
+   both halves. Promote any "OUT-OF-LANE FLAG" that the owning lens confirms.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/_lib/review-subagent-pattern.md]: Valid. Clarified in _lib that the parallel reviewers cannot see each other's output, so OUT-OF-LANE FLAG promotion happens during the caller's synthesis, where the caller checks each flag against the counterpart reviewer's mandate and findings before promoting. (9026c631)

File: .claude/commands/_lib/review-subagent-pattern.md [hunk 3]
```diff
@@ -50,3 +54,38 @@ After receiving findings:
...
+1. **Partition the mandate, not just the label.** Two reviewers with the same scope
+   but different titles mostly duplicate each other — you pay N× tokens for largely
+   the same output. Each panel persona must get: (a) a **mandate** = the dimensions
+   it owns and must go deep on; (b) an explicit **out-of-scope** list = the
+   dimensions its counterpart owns, which it must NOT report except as one-line
+   "OUT-OF-LANE FLAGS" at the end. Route the CRITERIA items to the reviewer whose
+   mandate covers them (an item may be shared only when both lenses genuinely apply).
+2. **Spawn all panel reviewers in parallel** — in a single message with multiple
+   Agent tool calls — so wall-clock does not increase. Each prompt is self-contained
+   (embed FILES as absolute paths; copy that reviewer's CRITERIA subset verbatim;
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/_lib/review-subagent-pattern.md]: Valid. Reconciled _lib with the command usage: it now documents both (a) pre-routed CRITERIA subset and (b) shared-floor-with-filtering, and notes mkarch/mkplan use (b). The old "copy the subset verbatim" wording implied only (a). (9026c631)

File: .claude/commands/mkplan.md [hunk 4]
```diff
@@ -67,13 +67,16 @@ Work in the following order.
...
+     - **Reviewer B — test / test-infra (SDET).** Mandate (go deep): every AC has at least one concrete test/static verification task and an explicit AC-verification section (AC→test coverage); tests cover non-trivial logic, error paths, boundary values; creation tasks have ground-truth verification; no unjustified duplicate tests; test-helper files follow the test-organization guide; AND all **Conditional checks** (rerun isolation, normalization helpers, narrow linter-suppression scoping, CI scripted-logic extraction, env-skip purity, resource cleanup/leak at acquisition, repeated-CLI startup validation). Out-of-scope (defer to Reviewer A — one-line OUT-OF-LANE FLAGS): production-code task decomposition, file paths, phase ordering, reuse of production code.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid — test code (*_test.go, testutil/) is Go source but falls in Reviewer B's lane, so the no-Japanese rule could slip through. Scoped Reviewer A to production Go source and added the no-Japanese-in-Go-test-code rule to Reviewer B's mandate. (9026c631)

File: .claude/commands/mkplan.md [hunk 5]
```diff
@@ -67,13 +67,17 @@ Work in the following order.
...
+     - **Reviewer B — test / test-infra (SDET).** Mandate (go deep): every AC has at least one concrete test/static verification task and an explicit AC-verification section (AC→test coverage); tests cover non-trivial logic, error paths, boundary values; creation tasks have ground-truth verification; no unjustified duplicate tests; test-helper files follow the test-organization guide; **no Japanese in Go test code** (`*_test.go` and `testutil/` helpers are Go source too, and fall in B's lane not A's); AND all **Conditional checks** (rerun isolation, normalization helpers, narrow linter-suppression scoping, CI scripted-logic extraction, env-skip purity, resource cleanup/leak at acquisition, repeated-CLI startup validation). Out-of-scope (defer to Reviewer A — one-line OUT-OF-LANE FLAGS): production-code task decomposition, file paths, phase ordering, reuse of production code.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. The phase-order item (Conditional check) now belongs to Reviewer A explicitly: A owns "phase names/order match architecture", and B owns all Conditional checks EXCEPT the phase-order item. No longer falls between lanes. (5f43b58d)

[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Reviewer B's no-Japanese-in-Go-test scope now includes package-internal helpers test_helpers.go / test_helpers_<category>.go (//go:build test), in addition to *_test.go and testutil/. (5f43b58d)

File: .claude/commands/_lib/review-subagent-pattern.md [hunk 6]
```diff
@@ -14,7 +14,11 @@ This is project-independent. It depends on nothing in `_context.md`.
   "this phase group's code changes", "the translation").
 - **PERSONA** — the reviewer role and focus (e.g. "an experienced software
   architect and senior SRE", "a senior Go engineer and senior SRE", "a technical
-  translator and editor").
+  translator and editor"). This may be **a single persona** (single-reviewer mode,
+  the default) or **a panel of two-or-more partitioned personas** (panel mode — see
+  "Panel mode" below). A panel persona is specified as a role PLUS a non-overlapping
+  **mandate** (what it owns) and an explicit **out-of-scope** list (what its
+  counterpart owns).
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/_lib/review-subagent-pattern.md]: Valid. Clarified: out-of-scope is what "the other panel personas own" (the counterpart for a two-reviewer panel), consistent with the "two-or-more" framing. (5f43b58d)

File: .claude/commands/_lib/review-subagent-pattern.md [hunk 7]
```diff
@@ -50,3 +54,45 @@ After receiving findings:
...
+
+A calling command opts into panel mode by supplying a PERSONA that is a panel
+(two-or-more partitioned personas). When PERSONA is a panel:
+
+1. **Partition the mandate, not just the label.** Two reviewers with the same scope
+   but different titles mostly duplicate each other — you pay N× tokens for largely
+   the same output. Each panel persona must get: (a) a **mandate** = the dimensions
+   it owns and must go deep on; (b) an explicit **out-of-scope** list = the
+   dimensions its counterpart owns, which it must NOT report except as one-line
+   "OUT-OF-LANE FLAGS" at the end. Route the CRITERIA items to the reviewer whose
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/_lib/review-subagent-pattern.md]: Valid. Step 1 now says the out-of-scope dimensions are owned by "the other panel personas (collectively, when the panel has more than two)". (5f43b58d)

File: .claude/commands/_lib/review-subagent-pattern.md [hunk 8]
```diff
@@ -50,3 +54,45 @@ After receiving findings:
...
+   OUT-OF-LANE FLAGS. Shared floor needs no item-by-item routing and avoids coverage
+   gaps from mis-routing; pre-routing minimizes each reviewer's reading.
+3. **Synthesize before fixing.** The caller (not a subagent) merges the panel
+   outputs: **dedup** overlapping findings; **reconcile** conflicting severities
+   (take the higher); and **reconstruct cross-cutting issues** that sit in the seam
+   between mandates (a structural choice with an operational consequence) — this
+   synthesis is panel mode's main risk to mitigate, since the parallel reviewers
+   cannot see each other's output. For each "OUT-OF-LANE FLAG", the caller checks it
+   against the counterpart reviewer's mandate and findings during this synthesis and
+   promotes it to a real finding if it holds up.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/_lib/review-subagent-pattern.md]: Valid. Step 3 now validates an OUT-OF-LANE FLAG against "the mandate and findings of the panel persona(s) that own that dimension", not a single counterpart. (5f43b58d)

File: .claude/commands/mkarch.md [hunk 9]
```diff
@@ -52,11 +52,15 @@ Work in the following order.
...
+     - **Reviewer B — senior SRE.** Mandate (go deep here, and apply it EVEN WHERE THE CHECKLISTS ARE SILENT — these dimensions are mostly not enumerated below and are exactly where this reviewer adds value): failure modes and fail-closed vs fail-open correctness; TOCTOU / race windows between risk evaluation and actual use; production security posture, blast radius, and evasion/bypass paths; auditability/observability for on-call (can a denial be explained from the logs?); rollout/migration safety and backward compatibility (what breaks on upgrade; is a staged/shadow rollout needed?); performance/latency and pathological inputs; determinism/reproducibility (environment-dependent results, dry-run vs runtime); debuggability and safe override. Out-of-scope (defer to Reviewer A — at most one-line "OUT-OF-LANE FLAGS"): interface/type design, package boundaries/DRY, naming/terminology, Mermaid conventions, abstraction structure, AC-traceability bookkeeping.
+     - Each reviewer must find real problems, not approve, and must stay in its lane (the partition is what makes the panel worth its ~2× cost).
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkarch.md]: Valid. Reworded mkarch to "look hard for real problems and not rubber-stamp ... if a lane is genuinely clean, it says so explicitly rather than inventing findings", to avoid encouraging false positives. (5f43b58d)

File: .claude/commands/mkplan.md [hunk 10]
```diff
@@ -67,13 +67,17 @@ Work in the following order.
...
-   - **CRITERIA**: every item from the Technical correctness checklist, the Conditional checks section, and the Readability and consistency checklist below, copied verbatim. For Conditional checks, evaluate each item that applies to the plan; mark inapplicable items N/A (N/A is not a finding).
-
-   Extra rule: commit the implementation plan document only after all review passes are complete and all Critical and Major issues are resolved.
+   - **PERSONA**: default to a **single reviewer** — an experienced senior engineer and senior SRE — for routine plans; the plan checklists below are exhaustive enough that a single reviewer rarely satisfices. Use **panel mode** (per `_lib`, two partitioned reviewers run in parallel) only for **high-risk or large plans** — e.g. a heavy integration-test / CI / external-resource surface, or a security-gate / migration plan (staged rollout, simultaneous behavior raises and lowers, many test updates). When panel mode is warranted, partition by **what to build vs. how to verify it** (the natural seam of a plan), NOT engineer-vs-SRE (those overlap on a plan):
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Replaced "rarely satisfices" with a direct statement ("a single reviewer is sufficient for routine plans, because the checklists are already exhaustive...") and referenced the specific _lib/review-subagent-pattern.md file. (5f43b58d)

File: .claude/commands/mkplan.md [hunk 11]
```diff
@@ -67,13 +67,17 @@ Work in the following order.
...
+     - **Reviewer A — implementation (SWE).** Mandate (go deep): every AC is addressed by ≥1 implementation task (AC→impl coverage); reuse / DRY vs re-implementation; file paths and package-boundary fit; phase order matches the approved architecture's priorities; external-assumption verification (tool-flag semantics, real API/lifecycle-method existence, image/CI schema); no Japanese in **production** Go source; the plan references architecture instead of duplicating it. Out-of-scope (defer to Reviewer B — one-line OUT-OF-LANE FLAGS): test coverage/quality, test-infra isolation, helper-file organization, the Conditional checks.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Aligned Reviewer A's wording with step 6: "English-only (no Japanese) in production Go comments, identifiers, and string literals (per step 6)". (5f43b58d)

File: .claude/commands/mkplan.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Aligned Reviewer B's wording with step 6 ("the same English-only rule (no Japanese in comments, identifiers, or string literals) applied to Go test source"), with the parenthetical focused on which files are covered. (5f43b58d)

File: .claude/commands/mkplan.md [hunk 12]
```diff
@@ -67,13 +67,17 @@ Work in the following order.
...
-
-   Extra rule: commit the implementation plan document only after all review passes are complete and all Critical and Major issues are resolved.
+   - **PERSONA**: default to a **single reviewer** — an experienced senior engineer and senior SRE. A single reviewer is sufficient for routine plans, because the plan checklists below are already exhaustive and leave little room for a combined reviewer to under-cover one lens. Use **panel mode** (per `_lib/review-subagent-pattern.md`, two partitioned reviewers run in parallel) only for **high-risk or large plans** — e.g. a heavy integration-test / CI / external-resource surface, or a security-gate / migration plan (staged rollout, simultaneous behavior raises and lowers, many test updates). When panel mode is warranted, partition by **what to build vs. how to verify it** (the natural seam of a plan), NOT engineer-vs-SRE (those overlap on a plan):
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Use the fully-qualified .claude/commands/_lib/review-subagent-pattern.md path consistently in this step (no bare _lib). (3a9aefe9)

File: .claude/commands/mkplan.md [hunk 13]
```diff
@@ -67,13 +67,17 @@ Work in the following order.
...
+   - **CRITERIA**: every item from the Technical correctness checklist, the Conditional checks section, and the Readability and consistency checklist below, copied verbatim. For Conditional checks, evaluate each item that applies to the plan; mark inapplicable items N/A (N/A is not a finding). In panel mode, give both reviewers all of them as the shared floor (option (b) in `_lib`), but each reports only items within its mandate (deferring the rest as one-line OUT-OF-LANE FLAGS); the Conditional checks are Reviewer B's, except the phase-names/order item which is Reviewer A's (it is an architecture-alignment concern in A's mandate).
+   - **Synthesis** (panel mode only): after the parallel panel returns, YOU (not a subagent) merge the two outputs per `_lib` panel-mode step 3 — dedup overlapping findings, reconcile conflicting severities to the higher, reconstruct the full AC→{task,test} matrix from both reviewers' AC checks (so no AC is dropped from either the build or the verify side), and promote any confirmed OUT-OF-LANE FLAG. Then run the fix / re-review loop on the merged findings.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Replaced "option (b) in _lib" / "per _lib panel-mode step 3" with the fully-qualified file path and the named "Synthesize" step. (3a9aefe9)

File: .claude/commands/mkarch.md [hunk 14]
```diff
@@ -52,11 +52,15 @@ Work in the following order.
...
+   - **FILES**: the architecture document, the requirements document, the requirements process guide, and the Mermaid reference guide (paths in `_context.md`), as resolved absolute-path strings. If the conditional-guide trigger applies (`_context.md`, Domain-specific), also include the conditional guide. Pass the same FILES to both reviewers.
+   - **CRITERIA**: give BOTH reviewers every item from the Technical correctness checklist and the Readability and consistency checklist below, copied verbatim, as the shared floor — but each reviewer reports only items within its mandate (deferring the rest as one-line OUT-OF-LANE FLAGS). Reviewer B additionally applies its operational mandate above beyond the checklists.
+   - **Synthesis**: after the parallel panel returns, YOU (not a subagent) merge the two outputs per `_lib` panel-mode step 3 — dedup overlapping findings, reconcile conflicting severities to the higher, and reconstruct any cross-cutting issue sitting in the seam between the two mandates (a structural choice with an operational consequence). Then run the fix / re-review loop on the merged findings.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkarch.md]: Valid. mkarch now references .claude/commands/_lib/review-subagent-pattern.md and the named "Synthesize" step instead of "per _lib panel-mode step 3". (3a9aefe9)

File: .claude/commands/_lib/review-subagent-pattern.md [hunk 15]
```diff
@@ -50,3 +54,46 @@ After receiving findings:
...
+   mandate covers them (an item may be shared only when both lenses genuinely apply).
+2. **Spawn all panel reviewers in parallel** — in a single message with multiple
+   Agent tool calls — so wall-clock does not increase. Each prompt is self-contained
+   (embed FILES as absolute paths; copy that reviewer's CRITERIA verbatim;
+   state mandate + out-of-scope; require the same Severity/Location/Problem/Suggestion
+   output format). Two ways to give CRITERIA — the calling command picks one and is
+   consistent: (a) **pre-routed** — give each reviewer only the checklist subset its
+   mandate owns; or (b) **shared floor** — give every reviewer the full checklist and
+   have each report only the items within its mandate, deferring the rest as one-line
+   OUT-OF-LANE FLAGS. Shared floor needs no item-by-item routing and avoids coverage
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/_lib/review-subagent-pattern.md]: Valid. Clarified in _lib: a reviewer raises an out-of-mandate item ONLY as a one-line OUT-OF-LANE FLAG when it actually spots a potential issue there; it does not enumerate clean out-of-mandate checklist items (silence = no concern). Avoids verbose/token-wasteful output. (3a9aefe9)

File: .claude/commands/mkarch.md [hunk 16]
```diff
@@ -52,11 +52,15 @@ Work in the following order.
...
+     - Each reviewer must look hard for real problems and not rubber-stamp, and must stay in its lane (the partition is what makes the panel worth its ~2× cost). If a lane is genuinely clean, it says so explicitly rather than inventing findings.
+   - **FILES**: the architecture document, the requirements document, the requirements process guide, and the Mermaid reference guide (paths in `_context.md`), as resolved absolute-path strings. If the conditional-guide trigger applies (`_context.md`, Domain-specific), also include the conditional guide. Pass the same FILES to both reviewers.
+   - **CRITERIA**: give BOTH reviewers every item from the Technical correctness checklist and the Readability and consistency checklist below, copied verbatim, as the shared floor — but each reviewer reports only items within its mandate (deferring the rest as one-line OUT-OF-LANE FLAGS). Reviewer B additionally applies its operational mandate above beyond the checklists.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkarch.md]: Valid. Same clarification applied to mkarch's CRITERIA bullet. (3a9aefe9)

File: .claude/commands/mkplan.md [hunk 17]
```diff
@@ -67,13 +67,17 @@ Work in the following order.
...
+     - The **seam both reviewers cover** is AC traceability — A checks AC→implementation-task, B checks AC→test-task — so the merged AC→{task,test} matrix catches any dropped AC from both ends; reconcile it in synthesis.
+   - **FILES**: the implementation plan document, the architecture document, the requirements document, the requirements process guide, and the test organization guide (paths in `_context.md`), as resolved absolute-path strings. In panel mode, pass the same FILES to both reviewers.
+   - **CRITERIA**: every item from the Technical correctness checklist, the Conditional checks section, and the Readability and consistency checklist below, copied verbatim. For Conditional checks, evaluate each item that applies to the plan; mark inapplicable items N/A (N/A is not a finding). In panel mode, give both reviewers all of them as the shared floor (option (b) in `_lib`), but each reports only items within its mandate (deferring the rest as one-line OUT-OF-LANE FLAGS); the Conditional checks are Reviewer B's, except the phase-names/order item which is Reviewer A's (it is an architecture-alignment concern in A's mandate).
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Same clarification applied to mkplan's CRITERIA bullet. (3a9aefe9)

File: .claude/commands/mkplan.md [hunk 18]
```diff
@@ -67,13 +67,17 @@ Work in the following order.
...
+     - **Reviewer B — test / test-infra (SDET).** Mandate (go deep): every AC has at least one concrete test/static verification task and an explicit AC-verification section (AC→test coverage); tests cover non-trivial logic, error paths, boundary values; creation tasks have ground-truth verification; no unjustified duplicate tests; test-helper files follow the test-organization guide; **the same English-only rule (no Japanese in comments, identifiers, or string literals) applied to Go test source** — `*_test.go`, `testutil/` helpers, AND package-internal test helpers `test_helpers.go` / `test_helpers_<category>.go` (`//go:build test`), which are Go source too and fall in B's lane not A's; AND all **Conditional checks except the phase-order item** (rerun isolation, normalization helpers, narrow linter-suppression scoping, CI scripted-logic extraction, env-skip purity, resource cleanup/leak at acquisition, repeated-CLI startup validation). Out-of-scope (defer to Reviewer A — one-line OUT-OF-LANE FLAGS): production-code task decomposition, file paths, phase ordering, reuse of production code.
```
[the remark this answers is not in the document]
Comment (isseis) [reply .claude/commands/mkplan.md]: Valid. Reviewer B now owns the entire Conditional section except the phase-order item (the inline list is explicitly illustrative, not exhaustive — it previously omitted the cleanup/close-API and build-tag checks). The cleanup/close/logout-API-existence check is split by lane: production-code lifecycle is Reviewer A's, test-code lifecycle is Reviewer B's — removing the boundary conflict. (3a9aefe9)
